app/controllers/users.js

```diff
@@ -83,12 +94,31 @@ exports.signout = (req, res, next) => {
   });
 };
 
-exports.list = (req, res, next) => {
-  const limit = req.query.limit || 5;
-  const offset = req.query.offset * limit || 0;
-  userService.listAll(limit, offset).then((users) => {
-    res.jsonp(users);
-  }).catch((err) => {
-    next(err);
+exports.createAdmin = (req, res, next) => {
+  const newUser = req.body;
```

Comment:
We are having some code repetition with the `signup` method, maybe there is some way to avoid it.